src/liballoc/boxed.rs

```diff
@@ -270,6 +270,20 @@ impl<T: ?Sized> Box<T> {
         // additional requirements.
         unsafe { Pin::new_unchecked(boxed) }
     }
+
+    /// Converts a `Box<T>` into a `Box<ManuallyDrop<T>>`
+    #[unstable(feature = "box_manually_drop", issue = "0")]
```

Comment:
I meant local helper functions in the `P` module... this also works but now you have to create a tracking issue and add it here.